src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -587,6 +613,18 @@ internal AggregatingLogger InitializeLogger(AnalyzeOptionsBase analyzeOptions)
             if (filePath != null)
             {
                 context.TargetUri = new Uri(filePath);
+
+                if (_computeHashes)
+                {
+                    context.Hashes = HashUtilities.ComputeHashes(filePath);
+                    if (_pathToHashDataMap != null)
```

Comment:
>_pathToHashDataMap [](http://example.com/codeflow?start=24&length=18)

`_pathToHashDataMap` and `_run` are only initialized when we are outputting to a file/stream using `SarifLogger`. This check will prevent generating a `NullRefException` when we are not outputting as a file/stream